rest-api/edit-conversation-image-title/server/server.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+import dotenv from 'dotenv';
+
```

Comment:
I also had to install `node-fetch`

Reply:
Hey @keerlu, 
Not sure why you had to install node-fetch? The `fetch` API is already a part of Javascript out-of-the-box, so we don't really need to use any other dependencies to make requests.

Reply:
Fetch is only available natively in node 18+ which is quite new

Reply:
Ah, I do indeed have an old version of Node (I had to downgrade at a previous job for reasons, and then forgot I'd done that). So that explains my problem, and I'll upgrade. But as Steven says it would be worth including anyway for others with older versions.